src/renderer/components/Settings.tsx

```diff
@@ -153,6 +153,43 @@ export default function SettingsUi() {
                 </>
             )}
 
+            <Forms.FormTitle>Custom Spash Animation</Forms.FormTitle>
```

Comment:
```suggestion
            <Forms.FormTitle>Custom Splash Animation</Forms.FormTitle>
```

Reply:
This got fixed in the latest commit.
![image](https://github.com/Vencord/Vesktop/assets/71154407/c3e04896-e88a-404b-bdc6-7646e8840487)